client/src/views/products/ProductListView.vue

```diff
@@ -101,7 +101,7 @@ const toggleDropdown = () => {
 </template>
 
 <style scoped>
->>> {
+:deep() {
```

Comment:
May be child class should be specified to avoid error like on this screenshot?

![image](https://user-images.githubusercontent.com/80485151/216899427-4c7662fa-4eb4-4406-bb11-74b908e4dee5.png)

Check this out
https://stackoverflow.com/questions/48032006/how-do-i-use-deep-or-or-v-deep-in-vue-js